src/usr/local/www/system_camanager.php

```diff
@@ -115,9 +115,7 @@
 		$pconfig['serial'] = $thisca['serial'];
 		$pconfig['trust']  = ($thisca['trust'] == 'enabled');
 		$pconfig['randomserial']  = ($thisca['randomserial'] == 'enabled');
-		if (!empty($thisca['prv'])) {
-			$pconfig['key'] = base64_decode($thisca['prv']);
-		}
+		$pconfig['key'] = base64_decode($thisca['prv']);
```

Comment:
Why change this test? The key could be empty when it's being read from the config, and if so, that `base64_decode()` will fail.

Reply:
Right, fixed